_includes/langselect.html

```diff
@@ -7,10 +7,11 @@
       {% else %}
         {% if lng == site.default_lang %}
           {% capture langlink %} {{ page.url }}{% endcapture %}
+          <li><a href="{{ langlink }}" rel="canonical" hreflang="{{ lng }}" data-slang="{{ lng }}">{{ lng }}</a></li>
         {% else %}
           {% capture langlink %}/{{ lng }}{{ page.url }}{% endcapture %}
+           <li><a href="{{ langlink }}"  rel="alternate" hreflang="{{ lng }}" data-slang="{{ lng }}">{{ lng }}</a></li>
         {% endif %}
-        <li><a href="{{ langlink }}" data-slang="{{ lng }}">{{ lng }}</a></li>
```

Comment:
and then
```suggestion
        <li><a href="{{ langlink }}"  rel="{{ lang_rel  }}" data-slang="{{ lng }}">{{ lng }}</a></li>
```

Reply:
What does "Applying suggestions on deleted lines is currently not supported" in GitHub web UI mean?

Reply:
It means you'll have to make the change yourself, rather than have GitHub do it for you.